easybuild/easyconfigs/i/intel/intel-2015b.eb

```diff
@@ -9,7 +9,7 @@ description = """Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 compver = '2015.3.187'
-gccsuff = '-GNU-4.9.3-2.25'
+gccsuff = '-GCCcore-4.9.3-binutils-2.25'
```

Comment:
You mean a dependency directly on GCC? Or just a different name? Definitely don't agree on depending in GCC directly since that wrecks the whole concept of having GCC/iccifort/PGI in the same `family`